### PR TITLE
Added proper logging levels for inframanager.

### PR DESCRIPTION
**Pull request by venkateshpnv**
The user can choose the log message level by setting the log level in the config.yaml file.

Signed-off-by: Venkatesh Petla <neelakanta.venkatesh.petla@intel.com>

### CODE REVIEW

File: inframanager/config.yaml [hunk 1]
```diff
@@ -12,7 +12,15 @@ p4BinPath: k8s_dp/k8s_dp.pb.bin
...
 P4ProgConf: k8s_dp/k8s_dp.conf
 HostName: "Node1"
-LogLevel: "Info"
+# Possible debug levels
+#  "Panic"
+#  "Fatal"
+#  "Error"
+#  "Warn"
+#  "Info"
+#  "Debug" (Default)
```
Comment (dhruvpt) [inline inframanager/config.yaml:21]: While for now, debug as default is fine. But in general, the default should be Warn or Error.

**Review by dhruvpt — APPROVED** [review]

File: inframanager/config.yaml [hunk 2]
```diff
@@ -12,7 +12,15 @@ p4BinPath: k8s_dp/k8s_dp.pb.bin
...
+#  "Trace"
+LogLevel: "Debug"
```
Comment (vishala-r) [inline inframanager/config.yaml:23]: what API is debug associated with? i mean log.Infof is debug? or info?

**Review by vishala-r — APPROVED** [review]

**Review by nupurjai — APPROVED** [review]

File: inframanager/config.yaml (hunk 2 above)
Comment (nupurjai) [reply inframanager/config.yaml:23]: This is universal log level

File: inframanager/config.yaml (hunk 1 above)
Comment (venkateshpnv) [reply inframanager/config.yaml:21]: Ok. We took the same level from the infraagent

File: inframanager/config.yaml (hunk 2 above)
Comment (venkateshpnv) [reply inframanager/config.yaml:23]: Below are the APIs associated for corresponding log levels.
log.Trace("Something very low level.")
log.Debug("Useful debugging information.")
log.Info("Something noteworthy happened!")
log.Warn("You should probably take a look at this.")
log.Error("Something failed but I'm not quitting.")
// Calls os.Exit(1) after logging
log.Fatal("Bye.")
// Calls panic() after logging
log.Panic("I'm bailing.")

**Pull request merged**